app/helpers/course/assessment/answer/programming_helper.rb

```diff
@@ -84,6 +84,17 @@ def get_test_cases_and_results(test_cases_by_type, auto_grading)
     end
   end
 
+  # Return the bootstrap class for highlighting the test case row.
+  #
+  # @param [Course::Assessment::Answer::ProgrammingAutoGradingTestResult] test_case_result The
+  #   test case result.
+  # @return [String] 'bg-success text-success', 'bg-danger text-danger' or empty string
+  #   if there is no test_case_result.
+  def get_test_result_highlight(test_case_result)
```

Comment:
Would recommend `test_result_class` as that's the convention of some of the helper functions.

Also, we specify css classes with `[Array<String>]` rather than `String` with spaces. I suspect this is because it is easier to add/chain on more classes (as you just need to do `array + array` rather than having to consider the spaces in a string). So maybe can modify the function to reflect that.